### PR TITLE
Yury ortuno lesson 1 homework 2

### PR DESCRIPTION
**Pull request by yuryver**

### CODE REVIEW

File: yury-ortuno/lesson-1/homework-2-Car/src/Car.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/**
+ * Created by Yury Ortuno on 11/13/2018.
+ */
+public class Car {
+    private String mark;
```
Comment (J4rv1sX) [inline yury-ortuno/lesson-1/homework-2-Car/src/Car.java:5]: This field is probably not relevant for our problem

File: yury-ortuno/lesson-1/homework-2-Car/src/Car.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        isPowerOn = true;
+        return isPowerOn;
+    }
+
+    public boolean powerOff() {
+        isPowerOn = false;
+        return isPowerOn;
+    }
+
+    public void fillGas(double gas) {
```
Comment (J4rv1sX) [inline yury-ortuno/lesson-1/homework-2-Car/src/Car.java:42]: Gas should be added to actual gas quantity you have

**Review by J4rv1sX — CHANGES_REQUESTED** [review]

File: yury-ortuno/lesson-1/homework-2-Car/src/Car.java (hunk 2 above)
Comment (yuryver) [reply yury-ortuno/lesson-1/homework-2-Car/src/Car.java:42]: Done

File: yury-ortuno/lesson-1/homework-2-Car/src/Car.java (hunk 1 above)
Comment (yuryver) [reply yury-ortuno/lesson-1/homework-2-Car/src/Car.java:5]: I use the field to show results

**Pull request closed without merging**